### DISCUSSION TITLE
Do I have to mention Quarto's name in my blog or website or can I use it without any attribution required?

### DISCUSSION THREAD

**Discussion by muhaksim** [quarto-dev/quarto-cli#9251] [Q&A] [answered]
### Description

I want to use Quarto for blogging and potentially for other things.
Do I have to mention Quarto's name in my blog or website or can I use it without any attribution required?
I want to follow and do what is allowed while I also want to have my own thing without any other name mentioned in my blog or website.

What should I do if I want to build a blog or website with Quarto in terms of attribution and license?

**Comment by mcanouil** [comment]
See the FAQ: https://quarto.org/docs/faq/

**Comment by muhaksim** [comment]
I saw the FAQ. But my question is:
Since Quarto is MIT-licensed, Do I have to mention its name and attribute it in my blog or website?
Or can I just use it without adding its name to my blog, website or even in my repo?

**Reply by mcanouil** [reply]
The "attribute part" regarding any license is about using/modifying the code of the software, not about the content you are creating using the software.

If you write a document using Microsoft Word, do you attribute Microsoft?

This being said, it's always a good practice to credits the tools you use, usually done in the footer of website or in a dedicated page, but this is unrelated to software licensing.

**Comment by muhaksim** [comment]
I get your point but in this case, the platform is built on Quarto not just the content I have written using it. I am using its software.
So in summary: I can use Quarto in any form I want (even for commercial) without the need to mention its name in my website or github repo? The attribution is abbreciated but it is not a must. Am I right?

**Reply by mcanouil** [reply]
> the platform is built on Quarto not just the content I have written using it. I am using its software.

What does this means?

Are you building a software/tools that uses the source code of Quarto CLI? => You must follow the license (https://mit-license.org/). Note that if you use 1.3 or earlier version of Quarto CLI, the license is not MIT. MIT is the license for 1.4 and above.

Are you using Quarto CLI to create website, books, PDF, etc.? => you work is not a derivate or a modification of Quarto CLI.

**Comment by muhaksim** [comment]
I meant using it to create websites, book , PDF, etc. So, I guess in this case the answer is that I don't need to mention Quarto's name at all but that it is appreciated, right?

**Reply by mcanouil** [reply, marked as answer]
Yes.

PS: you can reply in the thread. There is no need to make a new post in the discussion which split the actual discussion.

**Reply by muhaksim** [reply]
Ok. Sorry. Thanks, Mickaël Canouil for your time. I really appreciate it.

**Reply by mcanouil** [reply]
You're welcome and there is nothing to be sorry or to apologise for.
